distribution/docker/slim/Dockerfile

```diff
@@ -11,4 +11,11 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" && \
     esac && \
     wget --no-verbose https://packages.timber.io/vector/${VECTOR_VERSION}/vector-${VECTOR_VERSION}-${ARCH}.deb && \
     dpkg -i vector-${VECTOR_VERSION}-${ARCH}.deb && \
-    rm -f vector-${VECTOR_VERSION}-${ARCH}.deb
\ No newline at end of file
+    rm -f vector-${VECTOR_VERSION}-${ARCH}.deb
+
+FROM debian:9-slim
+
+COPY --from=builder /usr/bin/vector /usr/bin/vector
+COPY --from=builder /etc/vector /etc
```

Comment:
if these are the example configs, do they belong in a "slim" container?

Reply:
Should we also convert from spaces to tabs to save on the bytes there? 😄 

https://www.youtube.com/watch?v=SsoOG6ZeyUI

Reply:
no I think moving to emacs is the right decision here.